tests/utils.sh

```diff
@@ -19,3 +19,24 @@ function getAccount () # u16 port, string addr
                   https://127.0.0.1:${1}/api/account/${2} \
                   2>/dev/null)
 }
+
+function getAccountWithBalance () # u16 port, string addr and expected balance
+{
+    if [ $# -ne 3 ]; then
+        die "3 arguments expected for getAccountWithBalance, not $#"
+    fi
+
+    balance=0
+    for i in $(seq 30)
+    do
+	    balance=$(getAccount $1 $2 | jq ".Balance" | sed 's/"//g')
+	    if [ "$balance" == "$3" ];then
+		    return 0
+	    fi
+	    sleep 0.3
```

Comment:
According to `man 1 sleep`:
![screen shot 2018-08-16 at 09 14 34](https://user-images.githubusercontent.com/2180215/44179976-e0ff5000-a134-11e8-8a46-b8c7ab2c5f7a.png)
So I'd rather have a `sleep 1` here.

Reply:
@Geod24 As you said, it is not bottleneck of this test, but there are subtle difference as a result.

Reply:
What differences ?

Reply:
@Geod24 Finish fast?! :)